src/GraphQL.Tests/Serialization/SystemTextJsonTests.cs

```diff
@@ -119,6 +143,12 @@ private class TestClass2
             public JsonElement variables { get; set; }
         }
 
+        private class TestClass3
+        {
+            public string query { get; set; }
```

Comment:
Try to check that `Query` and `Variables` are deserialized too.

Reply:
Done, and also added tests to verify case sensitivity behavior.  However, the default behavior differs between System.Text.Json and Newtonsoft.Json.  I opted to leave them as the default for the platform, as that would be expected behavior.  I did add additional xml comments.